Index: src/main/kotlin/Main.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import dataSource.CsvDataSource\r\nimport dataSource.utils.CsvParser\r\nimport interactor.CostOfLivingDataSource\r\nimport interactor.GetCityHasCheapestInternetConnectionInteractor\r\nimport interactor.GetHighestSalaryAverageCititesNamesInteractor\r\n\r\nfun main() {\r\n    val csvParser = CsvParser()\r\n    val dataSource: CostOfLivingDataSource = CsvDataSource(csvParser)\r\n\r\n    val getHighestSalaryAverageCities = GetHighestSalaryAverageCititesNamesInteractor(dataSource)\r\n    println(getHighestSalaryAverageCities.execute(limit = 10))\r\n    printSeparationLine()\r\n\r\n    val getCityHasCheapestInternetConnectionInteractor = GetCityHasCheapestInternetConnectionInteractor(dataSource)\r\n    println(getCityHasCheapestInternetConnectionInteractor.execute())\r\n\r\n}\r\nprivate fun printSeparationLine(){\r\n    print(\"\\n_______________________________\\n\")\r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/kotlin/Main.kt b/src/main/kotlin/Main.kt
--- a/src/main/kotlin/Main.kt	(revision e9222645a5eb90fb93cf804ba7c822d869cdbf8e)
+++ b/src/main/kotlin/Main.kt	(date 1675888318593)
@@ -8,6 +8,7 @@
     val csvParser = CsvParser()
     val dataSource: CostOfLivingDataSource = CsvDataSource(csvParser)
 
+
     val getHighestSalaryAverageCities = GetHighestSalaryAverageCititesNamesInteractor(dataSource)
     println(getHighestSalaryAverageCities.execute(limit = 10))
     printSeparationLine()
Index: build.gradle.kts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import org.jetbrains.kotlin.gradle.tasks.KotlinCompile\r\n\r\nplugins {\r\n    kotlin(\"jvm\") version \"1.6.20\"\r\n    application\r\n}\r\n\r\ngroup = \"org.the-chance\"\r\nversion = \"1.0-SNAPSHOT\"\r\n\r\nrepositories {\r\n    mavenCentral()\r\n}\r\n\r\ndependencies {\r\n    testImplementation(kotlin(\"test\"))\r\n    testImplementation(\"org.junit.jupiter:junit-jupiter:5.8.1\")\r\n    testImplementation(\"org.junit.jupiter:junit-jupiter:5.8.1\")\r\n    testImplementation(\"org.junit.jupiter:junit-jupiter:5.8.1\")\r\n}\r\n\r\ntasks.test {\r\n    useJUnitPlatform()\r\n}\r\n\r\ntasks.withType<KotlinCompile> {\r\n    kotlinOptions.jvmTarget = \"1.8\"\r\n}\r\n\r\napplication {\r\n    mainClass.set(\"MainKt\")\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build.gradle.kts b/build.gradle.kts
--- a/build.gradle.kts	(revision e9222645a5eb90fb93cf804ba7c822d869cdbf8e)
+++ b/build.gradle.kts	(date 1675976900833)
@@ -17,6 +17,7 @@
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
     testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
+    testImplementation("org.junit.jupiter:junit-jupiter:5.8.1")
 }
 
 tasks.test {
Index: src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt b/src/test/kotlin/interactor/GetHighestSalaryAverageCitiesNamesInteractTest.kt
rename from src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt
rename to src/test/kotlin/interactor/GetHighestSalaryAverageCitiesNamesInteractTest.kt
--- a/src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt	(revision e9222645a5eb90fb93cf804ba7c822d869cdbf8e)
+++ b/src/test/kotlin/interactor/GetHighestSalaryAverageCitiesNamesInteractTest.kt	(date 1675989892782)
@@ -1,12 +1,22 @@
 package interactor
 
+import org.junit.jupiter.api.BeforeAll
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class GetHighestSalaryAverageCititesNamesInteractor {
 
-import org.junit.jupiter.api.Assertions.*
+    lateinit var highestSalaryAverageCititesNamesInteractor: GetHighestSalaryAverageCititesNamesInteractor
 
-class GetCityHasCheapestInternetConnectionInteractorTest {
+    @BeforeAll
+    fun setUp(){
+        highestSalaryAverageCititesNamesInteractor = GetHighestSalaryAverageCititesNamesInteractor()
+    }
+
 
     @Test
-    fun execute() {
+    fun execute(){
+
     }
 }
\ No newline at end of file
